telemetry_subscribers/src/lib.rs

```diff
@@ -64,14 +68,49 @@ type ChromeGuard = tracing_chrome::FlushGuard;
 #[cfg(not(feature = "chrome"))]
 type ChromeGuard = ();
 
-pub struct TelemetryGuards(WorkerGuard, Option<ChromeGuard>);
+pub struct TelemetryGuards(Vec<WorkerGuard>, Option<ChromeGuard>);
+
+fn get_output(config: &TelemetryConfig) -> (BoxMakeWriter, Vec<WorkerGuard>) {
+    let cur_dir = std::env::current_dir().ok().unwrap().join("logs");
+    std::fs::create_dir_all(&cur_dir).ok().unwrap();
+    let (nb_stdout, nb_stdout_guard) = tracing_appender::non_blocking(std::io::stdout());
+    let (nb_stderr, nb_stderr_guard) = tracing_appender::non_blocking(std::io::stderr());
+
+    // default setting:
+    // (after filtering, controlled by env_filter)
+    // TRACE -> stdout
+    // DEBUG -> stdout
+    // INFO  -> stdout, stdout logfile (if enabled)
+    // WARN  -> stdout, stdout logfile (if enabled)
+    // ERROR -> stderr, stderr logfile (if enabled)
+
+    let mk_writer = nb_stdout
+        .with_max_level(Level::TRACE)
+        .with_min_level(Level::WARN)
+        .and(nb_stderr.with_max_level(Level::ERROR));
+
+    let mut guards = vec![nb_stderr_guard, nb_stdout_guard];
 
-fn get_output(config: &TelemetryConfig) -> (NonBlocking, WorkerGuard) {
     if let Some(logfile_prefix) = &config.log_file {
-        let file_appender = tracing_appender::rolling::daily("", logfile_prefix);
-        tracing_appender::non_blocking(file_appender)
+        let (nb_stdout_file, nb_stdout_file_guard) = tracing_appender::non_blocking(
+            tracing_appender::rolling::daily(&cur_dir, format!("{logfile_prefix}.stdout")),
+        );
+        guards.push(nb_stdout_file_guard);
+        let (nb_stderr_file, nb_stderr_file_guard) = tracing_appender::non_blocking(
+            tracing_appender::rolling::daily(&cur_dir, format!("{logfile_prefix}.stderr")),
+        );
+        guards.push(nb_stderr_file_guard);
+        let mk_writer = mk_writer
+            .and(nb_stderr_file.with_max_level(Level::ERROR))
+            .and(
+                nb_stdout_file
+                    .with_max_level(Level::INFO)
```

Comment:
if anything file logging should default to be more verbose than stdout, because files can easily be bigger and handle more writes, stdout is often used by people to look at what's happening and having too much STDOUT output overwhelms and makes debugging harder.